canonical-websites/www.ubuntu.com
# Smarter navigation

Replace the existing, somewhat confusing `level_1` etc. navigation system with a new one:

- A `navigation.yaml` defines the navigation sections, and the hierarchy of children within each
- The breadcrumb navigation will be displayed according to this hierarchy
- The main navigation in the header will display the sections in `navigation.yaml` that have `include_in_menu: True`
- Individual pages can override the default breadcrumbs etc. by using the new `header` block
  - I've used this so far only in `cloud/contact-us.html` and `cloud/thank-you.html`

## QA

`./run`

Browse to <http://0.0.0.0:8001/cloud>, for example. Click around and check everything looks accurate. Check navigation in header, breadcrumbs and footer See <http://0.0.0.0:8001/cloud/contact-us> and <http://0.0.0.0:8001/cloud/thank-you> as well.
